### PR TITLE
[Ryujinx.Input] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670

### CODE REVIEW

File: src/Ryujinx.Input/HLE/NpadManager.cs [hunk 1]
```diff
@@ -193,7 +196,7 @@ public void Update(float aspectRatio = 1)
 
                         inputState = controller.GetHLEInputState();
 
-                        inputState.Buttons |= _device.Hid.UpdateStickButtons(inputState.LStick, inputState.RStick);
+                        inputState.Buttons |= Hid.UpdateStickButtons(inputState.LStick, inputState.RStick);
```
Comment (AcK77) [inline src/Ryujinx.Input/HLE/NpadManager.cs:199]: This change is strange or maybe I misunderstood something. (Or maybe there is something wrong in our code)

File: src/Ryujinx.Input/Motion/CemuHook/Client.cs [hunk 2]
```diff
@@ -380,35 +381,33 @@ public void RequestInfo(int clientId, int slot)
 
             Header header = GenerateHeader(clientId);
 
-            using (MemoryStream stream = MemoryStreamManager.Shared.GetStream())
-            using (BinaryWriter writer = new BinaryWriter(stream))
-            {
-                writer.WriteStruct(header);
+            using MemoryStream stream = MemoryStreamManager.Shared.GetStream();
+            using BinaryWriter writer = new(stream);
```
Comment (AcK77) [inline src/Ryujinx.Input/Motion/CemuHook/Client.cs:385]:
```suggestion
            using BinaryWriter writer = new(stream);

```

File: src/Ryujinx.Input/Motion/CemuHook/Client.cs [hunk 3]
```diff
@@ -420,39 +419,37 @@ public unsafe void RequestData(int clientId, int slot)
...
```
Comment (AcK77) [inline src/Ryujinx.Input/Motion/CemuHook/Client.cs:423]:
```suggestion
            using BinaryWriter writer = new(stream);

```

**Review by AcK77 — CHANGES_REQUESTED** [review]

**Review by marysaka — APPROVED** [review]
same comment as Ack, otherwise lgtm.

**Review by marysaka — APPROVED** [review]

**Review by AcK77 — APPROVED** [review]

**Pull request merged**